File: iosApp/iosApp/Feature/Favourites/Components/FavouritesEmptyContent.swift
import SwiftUI

struct FavouritesEmptyContent: View {
    @Environment(\.colorScheme) private var colorScheme

    var body: some View {
        let palette = PosaPalette(colorScheme: colorScheme)

        VStack(spacing: 0) {
            Text("🐾")
                .font(.system(size: 48))

            Spacer().frame(height: 12)

            Text("No favourites yet")
                .font(.headline)
                .fontWeight(.bold)

            Spacer().frame(height: 6)

            Text("Swipe right on a breed to save it here.")
                .font(.footnote)
                .foregroundStyle(palette.onBackground.opacity(0.6))
        }
        .multilineTextAlignment(.center)
        .padding(32)
        .frame(maxWidth: .infinity, maxHeight: .infinity)
    }
}

File: androidApp/src/main/kotlin/io/posa/feature/favourites/component/FavouritesEmptyContent.kt
package io.posa.feature.favourites.component

import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.Spacer
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.height
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.LocalTextStyle
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.platform.testTag
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import io.posa.core.common.UiIdentifiers


@Composable
internal fun FavouritesEmptyContent() {
    Box(
        modifier = Modifier
            .fillMaxSize()
            .testTag(UiIdentifiers.FAVOURITES_EMPTY),
        contentAlignment = Alignment.Center,
    ) {
        Column(
            horizontalAlignment = Alignment.CenterHorizontally,
            modifier = Modifier.padding(32.dp),
        ) {
            Text(text = "🐾", style = MaterialTheme.typography.displayMedium)
            Spacer(modifier = Modifier.height(12.dp))
            Text(
                text = "No favourites yet",
                style = MaterialTheme.typography.titleMedium,
                fontWeight = FontWeight.Bold,
            )
            Spacer(modifier = Modifier.height(6.dp))
            Text(
                text = "Swipe right on a breed to save it here.",
                style = MaterialTheme.typography.bodySmall,
                color = LocalTextStyle.current.color.copy(alpha = 0.6f),
            )
        }
    }
}